internal/kafka/test/integration/cloudprovider_test.go

```diff
@@ -360,9 +363,18 @@ func TestListCloudProviderRegions(t *testing.T) {
 				Schedulable:           true,
 			},
 		})
+		dataPlaneClusterConfig = dc
 
 	})
 	defer teardown()
+	var clusterService services.ClusterService
+	h.Env.MustResolve(&clusterService)
+	cluster := dataPlaneClusterConfig.ClusterConfig.GetManualClusters()
+
+	for _, c := range cluster {
```

Comment:
let's make it clear that this is a slice
```suggestion
	clusters := dataPlaneClusterConfig.ClusterConfig.GetManualClusters()

	for _, c := range clusters {
```